### DISCUSSION TITLE
Sample for chatBot using Azure Communication Resources/UI Library.

### DISCUSSION THREAD

**Discussion by jorgecolon413** [Azure/communication-ui-library#3350] [Q&A]
I'm currently developing a chat application for both internal and external users. The application utilizes the Azure Communication Services (ACS) and a UI library to provide its functionality. We are contemplating the integration of OpenAI resources from Microsoft (MSFT) to incorporate a chatbot feature into the application, while ensuring that the existing user experience built with the ACS UI library remains intact.

Is there a sample or roadmap available that outlines how to use a user-like chatbot, powered by ChatGPT resources, in conjunction with the Azure Communication Services and UI library to maintain a seamless user experience?

**Comment by AmyL219** [comment]
We appreciate your interest in AI integration for your ACS Chat scenario. Currently, our team is heavily engaged in working towards this integration. However, we have yet to finalize a decision or define a timeline for the upcoming feature's implementation. In the meantime, we encourage you to explore the benefits and enhancements your Azure Communication Services could obtain by [utilizing the Azure AI Service](https://learn.microsoft.com/en-us/azure/communication-services/concepts/chat/concepts#build-intelligent-ai-powered-chat-experiences).
